server/index.js

```diff
@@ -46,14 +41,6 @@ app.use(
     },
   }),
 );
-app.use(

```

Comment:
Er don't we need this?

Reply:
Seems to work fine without it. Possibly because of the `connect()` call I added at the top.